### PR TITLE
Add changes to list

### PR DESCRIPTION
**Pull request by marcelgarus**
Closes: #46

### CODE REVIEW

File: packages/ChangesReloaded-Core.package/CRBrowser.class/instance/isInImage..st [hunk 1]
```diff
@@ -0,0 +1,11 @@
+accessing
+isInImage: aChangeRecord
+	"Compares the source code of aChangeRecord to the code that is currently in the image."
+	| class currentSource changeSource |
+	
+	class := aChangeRecord methodClass.
+	(class notNil and: [class includesSelector: aChangeRecord methodSelector])
```
Comment (gritfessel) [inline packages/ChangesReloaded-Core.package/CRBrowser.class/instance/isInImage..st:7]: `isNoClass` and `classHasSelector` or something like that

Comment (gritfessel) [reply packages/ChangesReloaded-Core.package/CRBrowser.class/instance/isInImage..st:7]: and a comment

File: packages/ChangesReloaded-Core.package/CRBrowser.class/instance/isInImage..st [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+		ifFalse: [^ false].
+	currentSource := (class sourceCodeAt: aChangeRecord methodSelector) asString.
```
Comment (gritfessel) [inline packages/ChangesReloaded-Core.package/CRBrowser.class/instance/isInImage..st:9]: `methodName` temp var

Comment (marcelgarus) [reply packages/ChangesReloaded-Core.package/CRBrowser.class/instance/isInImage..st:9]: Seems like the correct term for method names in Smalltalk is "selector".
That would also make it more consistent how we retrieve these values:
```smalltalk
class := aChangeRecord methodClass.
selector := aChangeRecord methodSelector.
```

File: packages/ChangesReloaded-Core.package/CRBrowser.class/instance/isInImage..st [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+isInImage: aChangeRecord 
+	"Compares the source code of aChangeRecord to the
+	code that is currently in the image."
+	
+	| class selector currentSource changeSource |
+	
+	class := aChangeRecord methodClass.
+	selector := aChangeRecord methodSelector.
+	
+	"Everything that does not modify a class or handles a method selector that is not in the respective class at the moment cannot be compared to actual code within the image. To be sure to catch all unmade changes, we defined these unverifiable changes as 'not in image'."
```
Comment (gritfessel) [inline packages/ChangesReloaded-Core.package/CRBrowser.class/instance/isInImage..st:11]:
```suggestion
	"Everything that does not modify a class or handle a method selector that is not in the respective class at the moment cannot be compared to actual code within the image. To be sure to catch all unmade changes, we defined these unverifiable changes as 'not in image'."
```

**Pull request merged**